src/state/actions/__test__/avgSalaries.test.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import * as rtl from '@testing-library/react';
+import configureStore from 'redux-mock-store';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import thunk from 'redux-thunk';
+import * as types from '../../types/index';
+
+import { getAvgSalaries } from '../avgSalaries';
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+const mock = new MockAdapter(axios);
+
+const avgSalaries = [
+  {
+    interest_id: 2,
```

Comment:
The `interest_id` only exists on the users DB but isn't currently being used in the app, so this test doesn't need that property anymore. Our current work on the salaries takes out this property. When the salaries branch has been merged, you can `git pull origin develop` and` knex migrate:down` your DB till you get to the last table, then redo `knex migrate:latest` and `knex seed:run` on both the backend development and testing databases, then start up your tests again.